### DISCUSSION TITLE
Multi-graph  !

### DISCUSSION THREAD

**Discussion by MehdiZouitine** [pyg-team/pytorch_geometric#4907] [Q&A] [answered]
Hello to all! Thank you for this beautiful library.
I would like to create a multi graph with only 1 type of node but several types of relationship. With the heterogeneous graph framework, we are obliged to declare several types of nodes.

Is it possible to create a graph with only one type of node but several types of edges (a multigraph)?

Thank you in advance!

**Comment by rusty1s** [comment, marked as answer]
You can either use `HeteroData` as well for this, or make use of an `edge_type` vector along your `edge_index` and use layers such as `RGCNConv` to process.

**Reply by MehdiZouitine** [reply]
Thanks you !! :D
